docs/getting-started.md

```diff
@@ -0,0 +1,101 @@
+# Getting Started
+
+## Prerequisites
+Before proceeding install the following requirements.
+- [Golang](https://go.dev/doc/install) (version 1.21.6)
+
+## Getting CoCos
+Get the cocos repository:
+`git clone https://github.com/ultravioletrs/cocos.git`
+
+### HAL
+Get the hardware abstraction layer from the [releases](https://github.com/ultravioletrs/cocos/releases) on the cocos repository. Two files will be required:
+- `rootfs.cpio.gz` - Initramfs
+- `bzImage` - Kernel
+
+Create two directories in `cocos/cmd/manager`, the directories are `img` and `tmp`.
+Copy the downloaded files to `cocos/cmd/manager/img`.
+
+## Starting manager server
+Manager is a gRPC client and needs gRPC sever to connect to. We have an example server for testing purposes in `test/manager-server`. Run the server as follows:
+
+`go run ./test/manager-server/main.go`
+
+the output should be simillar to this:
+`{"time":"2024-03-19T12:27:46.542638146+03:00","level":"INFO","msg":"manager_test_server service gRPC server listening at :7001 without TLS"}`
+
+## Running manager
+Next we need to start manager. But first we'll need to install some prerequisites.
+
+### Vsock
+This is used to communicate between manager and agent. To enable it run:
```

Comment:
Do I need to run this on the host? If yes, write. Also - give a reference (link) to the Vsock (for example https://wiki.qemu.org/Features/VirtioVsock and/or https://github.com/mdlayher/vsock)